packages/components/src/grid/types.ts

```diff
@@ -22,7 +22,7 @@ type GridColumns = ResponsiveCSSValue< number >;
 
 type GridRows = ResponsiveCSSValue< number >;
 
-export type Props = {
+export type GridProps = {
```

Comment:
Given the use of the `GridColumns` and `GridRows` types, the auto-generated docs in Storybook are less helpful. Could we add something to the `columns` and `rows` prop descriptions to illustrate that they are in effect `number | number[]`?

Nit: We should probably fix up the alphabetization of the props as well by correcting the order of `isInline` and `gap`.

Reply:
> Before even thinking about how to document this feature properly, we might have to first reconsider whether we want to continue providing this feature officially 🤔 Like, how useful are viewport breakpoints in a component library like this? Should we not even bother until container queries land?

Good points.

Personally, I would:

- see if this feature is used at all in the codebase
- discuss/agree if this needed by the component in the first place
- expand the conversation to the package level — ie., do we need such a mechanism in our package? If so, should we look into implementing a more defined pattern for responsive layouts at all?

Personally, this feature feels a bit overkill. And if we zoom out even further, I'd like to take a more broad look at layout components like `Flex`, `Grid` and `{H|V|Z}Stack`, to asses their APIs and many any potential adjustments.